python/pyspark/ml/wrapper.py

```diff
@@ -68,7 +68,7 @@ def _make_java_param_pair(self, param, value):
         """
         sc = SparkContext._active_spark_context
         param = self._resolveParam(param)
-        java_param = self._java_obj.getParam(param.name)
+        java_param = self._java_obj.getParam(param.name.rstrip("_"))
```

Comment:
Because "lambda" is key word in Python, so we use "lambda_" for substitution. We should change back to "lambda" when it passed to Java. Other parameters which also hit key words should also tackle like this.